internal/tiltfile/tiltfile_test.go

```diff
@@ -3136,6 +3136,26 @@ update_mode(UPDATE_MODE_MANUAL)
 	f.loadErrString("update_mode can only be called once")
 }
 
+func TestHelmSkipsTests(t *testing.T) {
```

Comment:
can you also add a test that asserts an error message if helm() is pointing to a bad directory?